### PR TITLE
Microverse project pull request

### PR DESCRIPTION
**Pull request by SSEKA**
An average simple html and css project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1>Hello Microverse</h1>
```
Comment (shella12) [inline index.html]: - [ ] **Project Requirement** 

> -  A .html file that includes a header with the text "Hello Microverse!". Use the h1 tag for that.  

:neckbeard: According to project requirements, you should add your `h1` element inside the `header` semantic tag. I suggest always using semantic tags in your HTML projects as semantics tags add accessibility 💯 to your app.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @SSEKA , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Correct use of git flow ✅ 
- `test.md` is added to `.gitignore` ✅ 
- Linter files have been added ✅ 

## Required Changes ♻️

- [x] Please 🙏  make sure that you have documented 🗒️  your project by adding a readme 🤓 that reflects 🪞 🪞  your project.  Use [this template](https://github.com/microverseinc/readme-template) provided by Microverse to add README as mentioned in the project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) 
> - Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- [x] Please ☺️  use descriptive commit messages. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). 😄 

Screenshot 2023-02-15 at 2 35 44 AM

- [ ] Please 🙏 🥺  add a description to your PR by adding a meaningful title and a short summary describing the changes ♻️  you have made in this PR. 

Screenshot 2023-02-15 at 2 30 04 AM

- [ ] While you made a great 👍 👍  effort adding linter files to your project you still have to fix your linter errors. Please fix your `stylelint` errors. You can do this manually or you can use the auto fix command in your terminal (it implies IF you HAVE installed linter checkers locally if not please use [these guidelines](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) to install them locally first):  
```sh 
npx stylelint "**/*.{css,scss}" —fix  
```  

Screenshot 2023-02-15 at 2 45 54 AM

**_Also check the comments under the review._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @SSEKA , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- Good use of .gitignore :+1: 
- Professional README file :+1: 

 ## Required Changes ♻️
- [ ] Kindly make sure to make to bring back linter checks as they are no longer running :recycle: 
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @SSEKA  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**